cmd/keycmd/list.go

```diff
@@ -612,7 +612,7 @@ func getEvmBasedChainAddrInfo(
 }
 
 func printAddrInfos(addrInfos []addressInfo) {
-	header := []string{"Kind", "Name", "Subnet", "Address", "Token", "Balance", "Network"}
+	header := []string{"Type", "Name", "Subnet", "Address", "Token", "Balance", "Network"}
```

Comment:
again, if we replace here, probably we want to replace all addIngo.kind stuff, or avoid replacing here